models/intel/icnet-camvid-ava-0001/description/icnet-camvid-ava-0001.md

```diff
@@ -6,7 +6,19 @@ A trained model of ICNet for fast semantic segmentation, trained on the CamVid\*
 
 The model input is a blob that consists of a single image of `1x3x720x960` in the BGR order. The pixel values are integers in the [0, 255] range.
 
-The model output for `icnet-camvid-ava-0001` is the predicted class index of each input pixel belonging to one of the 12 classes of the CamVid dataset.
+The model output for `icnet-camvid-ava-0001` is the predicted class index of each input pixel belonging to one of the 12 classes of the CamVid dataset:
+- Sky
+- Building
+- Pole
+- Road
+- Pavement
+- Tree
+- SignSymbol
+- Fence
+- Vehicle
+- Pedestrian
+- Bike
+- Unlabeled
```

Comment:
Maybe you could add this list as a file under `data/dataset_classes` instead?

Reply:
for segmentation models demo not using class labels, we only use mapping of classes to color and there is appropriate file with such mapping for CamVid but it is a bit different story

Reply:
OK.